	Exception
		프로그램이 실행 중에 에러가 발생하면 아무런 처리도 없이 강제 종료됨,
		프로그램을 강젤 종료가 되지 않게끔 처리하기 위해서,
		Exception을 이용해서 실행하는 도중에 에러가 발생하면 해당 에러를 잡아서 처리하고
		코드를 계속 이어서 작업하게끔 처리하는 방법
	try : 작업 영역, 이 영역에서는 Exception이 발생 할 수 있다.
	catch : try 영역에서 Exception이 발생 했을 때, 해당 Exception을 처리하는 영역
	finally : Exception이 발생하든 안하든 반드시 실행되는 영역
	throw : 원하는 부분에서 Exception을 강제로 발생시키는 행위
	throws : 발생한 Exception을 외부로 던지는 행위(메서드를 호출한 대상에게 Exception을 전달)